Update AGA network requirements for clarity (#1076)
neo4j/docs-aura: modules/ROOT/pages/graph-analytics/aga.adoc

=== Network requirements

> Aura Graph Analytics sessions use the Apache Arrow Flight protocol to communicate with the client you use to interact with the session — for example, the GDS Python client.

Reviewer: ```suggestion
Aura Graph Analytics sessions use the Apache Arrow Flight protocol to communicate with your client (for example, the GDS Python client that creates a session).
```

=== Network requirements

Aura Graph Analytics sessions use the Apache Arrow Flight protocol to communicate with the client you use to interact with the session — for example, the GDS Python client.

[NOTE]
====

> If you connect to Aura Graph Analytics from outside Aura (for example, running the GDS Python client on your own machine or infrastructure), you must allow outbound TCP traffic on port `8491` for Apache Arrow, in addition to the standard ports for your AuraDB instance. See xref:security/ip-addresses.adoc[Aura IP addresses and ports] for the full list of IP addresses and ports required.

Reviewer 2: ```suggestion
If you connect to Aura Graph Analytics from outside Aura (for example, running the GDS Python client on your own machine or infrastructure), you must allow outbound TCP traffic on port `8491` for Apache Arrow, in addition to the standard ports for your AuraDB instance. 
See xref:security/ip-addresses.adoc[Aura IP addresses and ports] for the full list of IP addresses and ports required.
```
